test/test_bin/test_check_mta_sts.py

```diff
@@ -0,0 +1,54 @@
+import unittest
+import boto3
+from botocore.exceptions import 
+from unittest.mock import patch
+
+import check_mta_sts
+
+class TestCheckMtaSts(unittest.TestCase):
+    
+    @patch('check_mta_sts.boto3.client')
+    def test_s3_initialization(self, mock_client):
+        mock_client.return_value =None
+        s3 = check_mta_sts.boto3.client('s3',
+                                        aws_access_key_id=check_mta_sts.AWS_ACCESS_KEY_ID,
+                                        aws_secret_access_key=check_mta_sts.AWS_SECRET_ACCESS_KEY,
+                                        aws_session_token=check_mta_sts.AWS_SESSION_TOKEN)
+        mock_client.assert_called_once_with('s3',
+                                            aws_access_key_id=check_mta_sts.AWS_ACCESS_KEY_ID,
+                                            aws_secret_access_key=check_mta_sts.AWS_SECRET_ACCESS_KEY,
+                                            aws_session_token=check_mta_sts.AWS_SESSION_TOKEN)
+        def test_domains(self):
```

Comment:
This test looks like it'd execute fine ✅

Ideally, we want to test the business logic of the code and not just static values - since these types of tests typically add minimal value and increase maintenance burdens 😔

You could expand this test to call the function under test and assert that the values are being passed processed as expected i.e. asserting that we call the S3 method with each value in the list 😁